include/core/shape.h

```diff
@@ -104,9 +109,12 @@ class Shape : public AutoDerivativeFunction<dim>
    * Return the volume displaced by the solid
    *
    * @param fluid_density The density of the fluid that is displaced
+   * @param pcout An optional output stream to display warnings regarding volume
+   * calculation
    */
   virtual double
-  displaced_volume(const double fluid_density) = 0;
+  displaced_volume(const double                        fluid_density,
```

Comment:
It's to print warnings when the displaced volume formula called is inexact, but since this function is only called when we integrate the particle motion, and we only do this with spheres, I can safely remove the warnings.